### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by timchenxiaoyu**
merge  some typo

### CODE REVIEW

**Comment by timchenxiaoyu** [comment]
I have sign the CLA

**Comment by timchenxiaoyu** [comment]
I had set email ,why?

**Comment by cblecker** [comment]
There are [six commits](https://github.com/kubernetes/kubernetes/pull/45945/commits) on your branch. You need to rebase the one with your actual changes on top of an updated master.

**Comment by timchenxiaoyu** [comment]
git status
On branch fix_typo
Your branch is up-to-date with 'origin/fix_typo'.


 git rebase master
Current branch fix_typo is up to date.

File: staging/src/k8s.io/client-go/plugin/pkg/client/auth/oidc/oidc.go [hunk 1]
```diff
@@ -216,9 +216,6 @@ type roundTripper struct {
 }
 
 func (r *roundTripper) RoundTrip(req *http.Request) (*http.Response, error) {
-	if len(req.Header.Get("Authorization")) != 0 {
```
Comment (jeffvance) [inline staging/src/k8s.io/client-go/plugin/pkg/client/auth/oidc/oidc.go]: Why are these lines deleted?

**Comment by jeffvance** [comment]
@k8s-bot test this
/approve
All changes are typo fixes except the one I commented on above. Otherwise looks good.

**Comment by yifan-gu** [comment]
@timchenxiaoyu Thanks for the PR, can you rebase?

File: staging/src/k8s.io/client-go/plugin/pkg/client/auth/oidc/oidc.go (hunk 1 above)
Comment (timchenxiaoyu) [inline staging/src/k8s.io/client-go/plugin/pkg/client/auth/oidc/oidc.go]: sorry，it's merge error.i don't intend to delete it.

**Comment by timchenxiaoyu** [comment]
I want to rebase ,but i don't know how to operate

**Comment by jeffvance** [comment]
Add release note or NONE

**Comment by timchenxiaoyu** [comment]
I hava the cla, why i miss cla label

**Comment by cblecker** [comment]
There are seven commits on your branch. You need to rebase the one with your actual changes on top of an updated master.

**Comment by timchenxiaoyu** [comment]
rebase failed
CONFLICT (content): Merge conflict in plugin/pkg/scheduler/scheduler.go
error: Failed to merge in the changes.
Patch failed at 0221 Scheduler should use shared informer for pods

**Comment by jeffvance** [comment]
add release note NONE to remove the "release-note-label-needed" label, eg:
```release-note
NONE
```

**Comment by jeffvance** [comment]
It must be frustrating that a small "fix typos in docs" pr has so much overhead. And maybe there is a way to skip the full CI process for simple doc fixes, but for now you'll need to rebase and re-push your changes.
I am not sure what else is needed for your cla. Perhaps @liggitt or @spiffxp can chime in?

**Comment by cblecker** [comment]
It's the merge commits. You can't merge master into your branch, you need to rebase your commit on top of master.

**Comment by spiffxp** [comment]
@timchenxiaoyu for CLA-related help try https://github.com/kubernetes/kubernetes/wiki/CLA-FAQ#help and if that doesn't work, try e-mailing helpdesk@rt.linuxfoundation.org

**Comment by timchenxiaoyu** [comment]
image

ths  @spiffxp ,I hava send mail to helpdesk@rt.linuxfoundation.org
